pkg/crypto/age.go

```diff
@@ -104,6 +104,20 @@ func Identity() (*age.X25519Identity, error) {
 	return generateIdentity(getAgePath())
 }
 
+func findMissingKeyForEmail(emails []string, keys []*api.PublicKey) []string {
+	if len(keys) == 0 {
+		return emails
+	}
+	emailSet := sets.NewString(emails...)
+	emailKeySet := sets.NewString()
+	for _, key := range keys {
```

Comment:
Would be more elegant to use a Map func here

Reply:
done